nx-X11/programs/Imakefile

```diff
@@ -1,8 +1,3 @@
-XCOMM $Xorg: Imakefile,v 1.4 2000/08/17 19:47:01 cpqbld Exp $
-
-
-
-XCOMM $XFree86: xc/programs/Imakefile,v 3.54 2003/04/14 20:37:16 herrb Exp $
```

Comment:
Uhm, I'd actually leave that in there as a reference for when it was last changed? The dates in there have been "helpful" to see that nomachine replaced some files with newer or older versions of the same file.